### ISSUE TITLE
Rule nesting with :local not working

### ISSUE THREAD

**Issue by silllli** [sveltejs/svelte-preprocess#238]
**Describe the bug**
I noticed that `:local` works when using it as a modifier, but not when nesting other selectors in it (using SASS)—as it does with `:global` and how I understood the docs:

> if you need to have some styles be scoped inside a global style tag, use :local in the same way you'd use :global.

**To Reproduce**
Add any rule nested in a `:local` rule.

This works:

```
:local(.center) {
  text-align: center;
}
```

Whereas this doesn’t:

```
:local {
  .center {
    text-align: center;
  }
}
```

**Expected behavior**
The rules nested in a `:local` rule to be local.

**Information about your project:**

- Operating system: macOS 10.15.6
- `svelte` version: 3.17.3
- `sapper` version: 0.28.0
- `svelte-preprocess` version: 4.2.0
- Project uses Rollup

**Comment by silllli** [comment]
As mentioned in #103.

**Comment by kaisermann** [comment]
Hey @silllli 👋 

Fixed in `4.5.2` 🎉

**Comment by silllli** [comment]
Great, thank you!
